### ISSUE TITLE
Monte Hall Problem Simulation using Python

### ISSUE THREAD

**Issue by rahulnarayaniitk** [animator/learn-python#580]
Hey @animator, 
I would like to raise an issue to start my open-source contribution journey with GSSoC '24.
I would like to simulate the very famous Manote-Hall Problem using Python.

**Objective of the project**: to simulate the problem and analyze the probability outcomes using Python.

Monty Hall’s problem comes from a famous movie where three doors are used to help you win a car. 
How? Each door hides something behind it–a car and two goats. Any door can have the car while the remaining two have goats. The probability of finding a car is ⅓. Now, if you select Door 1 and the host opens Door 3 to find a goat, your chances just become ⅔. 
So basically I will be writing a Python program to solve this problem.

@animator kindly assign me this issue to work on and assign a suitable level to it under GSSoC '24.

**Comment by animator** [comment]
This topic is not under the scope of this project. Pick another topic from Contributing Guide and raise a new issue.
